e2e/dpos-2-validators.toml

```diff
@@ -4,6 +4,13 @@
   All = true
   Condition = "contains"
   Expected = ["{{index $.NodeBase64AddressList 0}}", "{{index $.NodeBase64AddressList 1}}"]
+  Json = true
+  ExpectedJson = [
```

Comment:
The `Json` field is redundant, if `ExpectedJson` is not empty match it, otherwise don't.